packages/core/src/tracing/opentelemetry-instrumentations/restify.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const constants = require('../constants');
+
+module.exports.init = () => {
+  const { RestifyInstrumentation } = require('@opentelemetry/instrumentation-restify');
```

Comment:
_question:_ The test is disabled for Node.js 18, because the instrumentation does not work correctly there. Should we also skip loading the instrumentation on Node.js 18 then?